src/client/WebSocketClient.ts

```diff
@@ -364,11 +364,19 @@ export class WebSocketClient extends EventEmitter {
   private pingTime: number;
   private readonly pongTime: number;
 
+  private _subscriptions: {
+    [channel_name: string]: Required<WebSocketChannel>;
+  };
+
+  private _subscriptions_cache: WebSocketChannel[];
```

Comment:
This exists mainly for performance optimization so that every time ws.subscriptions is called we don't have to regenerate this array. It's hard to know if ws.subscriptions is going to be called more in the wild or ws.subscribe()/unsubscribe().